src/app/components/home/home.ts

```diff
@@ -9,13 +9,13 @@ import {Http} from 'angular2/http';
     styles: [require('./home.css')],
     template: require('./home.html')
 })
-export class Home {
+export class Home implements OnInit {
```

Comment:
This is recommended as discussed here: https://github.com/angular/angular/issues/5814